src/hooks/useGlobalAppState.ts

```diff
@@ -57,12 +64,14 @@ const onStateChange = (
  */
 
 export const useGlobalAppState = (): void => {
+  const isFirstStart = useRef(true)
```

Comment:
this looks like the removed `  const hasExecuted = useRef(false)	` no?

Reply:
Kind of, yes.

The logic is reversed, but it has the same goal of checking if this is the first run or not.

The only difference is that in previous code it was used to display a log (and before that it was used to prevent double navigation), in proposed code it is used to clear the AsyncStorage from `LastActivity` info.